bin/restic_backup.sh

```diff
@@ -108,7 +114,7 @@ restic backup \
 	--tag "$RESTIC_BACKUP_TAG" \
 	"${B2_ARG[@]}" \
 	"${exclusion_args[@]}" \
-	"${extra_args[@]}" \
```

Comment:
As my understanding is that `RESTIC_EXTRA_ARGS` should be applied to all restic commands, and `RESTIC_BACKUP_EXTRA_ARGS` only to the backup command, we should keep `"${extra_args[@]}" ` here